templates/kibana.yaml

```diff
@@ -56,8 +56,8 @@ spec:
           - kibana-prometheus-exporter,6.7.0,https://github.com/pjhampton/kibana-prometheus-exporter/releases/download/6.7.0/kibana-prometheus-exporter-6.7.0.zip
       extraContainers: |
         - name: initialize-kibana-index
-          image: mesosphere/kubeaddons-addon-initializer:v0.0.4
-          command: ["/bin/bash", "-c", "addon-initializer kibana && while true; do sleep 86400; done"]
+          image: mesosphere/kubeaddons-addon-initializer:v0.0.13
+          command: ["/bin/bash", "-c", "addon-initializer kibana && sleep infinity"]
```

Comment:
@joejulian @shaneutt  this logic should be part of the add-on initializer passing a flag or something `--wait`. Also, could you remind me why we need to wait forever here?

Reply:
Good point. I was only responding to the sleep loop but if this were an initContainer instead of a sidecar, exiting when finished would be ok. Is this possible, @gracedo?

https://github.com/helm/charts/blob/44d01d26c8b2b69066086b5788995680dfb6df40/stable/kibana/templates/deployment.yaml#L39-L46

Reply:
It does look like the version of Kibana we're using supports initContainers, so it should be able to be done (I guess we'll see if there was a reason why it wasn't done in the first place 🙃). However, this isn't technically related to this PR so I'm wary of making that change here. Would it be all right if we got this PR in first, and I'll make a follow-up PR for Kibana?

Reply:
https://jira.mesosphere.com/browse/DCOS-60303 for Kibana follow-up work